firestore/query.go

```diff
@@ -132,7 +132,17 @@ func (q Query) Where(path, op string, value interface{}) Query {
 // The op argument must be one of "==", "!=", "<", "<=", ">", ">=",
 // "array-contains", "array-contains-any", "in" or "not-in".
 func (q Query) WherePath(fp FieldPath, op string, value interface{}) Query {
-	proto, err := filter{fp, op, value}.toProto()
+	return q.WhereEntity(PropertyPathFilter{fp, op, value})
```

Comment:
nit: Naming the fields in the `PropertyPathFilter` would be nice.

Reply:
Done